lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -282,6 +288,20 @@ namespace :foreman_tasks do
       end
     end
 
+    def with_error_handling(task, what = _('task'))
```

Comment:
just FYI I THINK this will always be english I think, because i believe default params are evaluated on class evaluation, not sure and don't see the point of translation of task :)